loadgen/mlperf.conf

```diff
@@ -15,6 +15,7 @@ rnnt.*.performance_sample_count_override = 2513
 gptj.*.performance_sample_count_override = 13368
 llama2-70b.*.performance_sample_count_override = 24576
 stable-diffusion-xl.*.performance_sample_count_override = 5000
+rgat.*.performance_sample_count_override = 24576
```

Comment:
Should we increase this and the validation dataset size to a much larger number like for dlrm?